src/main/scala/regmapper/DescribedReg.scala

```diff
@@ -23,7 +24,14 @@ object DescribedReg {
     enumerations: Map[BigInt, (String, String)] = Map()): (T, RegFieldDesc) = {
     val rdesc = RegFieldDesc(name, desc, None, None,
       access, wrType, rdAction, volatile, reset.map{_.litValue}, enumerations)
+
     val reg = reset.map{i => RegInit(i)}.getOrElse(Reg(gen))
+    val byte = 0 // TODO
```

Comment:
why is this in here? The register description doesn't know or need to know about byte and offset.

Reply:
a `DescribedReg` may not even end up in a regmap call ever... I don't get what this is here for.